ThermofluidStream/Processes/FlowResistance.mo

```diff
@@ -41,15 +69,37 @@ some medium properties and the geometry of the pipe.
 
 protected
   SI.Density rho_in = max(rho_min, Medium.density(inlet.state))
-    "density of medium entering";
+    "Density of medium entering";
   SI.DynamicViscosity mu_in = Medium.dynamicViscosity(inlet.state)
-    "dynamic viscosity of medium entering";
+    "Dynamic viscosity of medium entering";
+  SI.Length perimeterActual "Actual perimeter of resistance";
+  SI.Area areaCrossActual "Actual area of resistance";
+  SI.Length D_h "Hydraulic diameter of resistance";
```

Comment:
`perimeterActual`, `areaCrossActual` and `D_h` seem could be parameters as well. I suppose this could simplify model translation. The equations below should be of course incorporated.

Reply:
Hmm true, as they are calculated before each simulation they could also be parameters. But then we have to give them start/default values to avoid "parameters don't have any value"-Error. Setting it to 0 seems fine?

Reply:
Do you think doing it like that is a good solution:
![image](https://user-images.githubusercontent.com/83768600/230347453-535dc909-3e14-4eaa-b65f-f9d63abadf9e.png)

(Still facing the problem, that we need to close with an 'else' statement)

Reply:
I think this is a good option. "else" could also be: `perimeterActual = ... else perimeter;` (so one line less, in fact)) and similarly for other params. If this is reasonable.

Reply:
Maybe `import TFS.Processes.Internal.ShapeOfResistance;` could shorten the lines for better readability.